abm/connector.py

```diff
@@ -203,6 +217,10 @@ def read_stream(self, catalog_file):
         #         for an Airbyte read operation
         streams = []
         for stream in self.catalog_dict['catalog']['streams']:
+            if stream['name']:
+               stream_name = self.get_stream_name()
+               if stream_name != "" and stream['name'] != stream_name:
```

Comment:
is there a scenario wherein `self.get_stream_name()` returns `""` ?

Reply:
Fixed Thanks